src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -85,7 +85,19 @@ mocha.describe('readings API', () => {
 					expectCompareToEqualExpected(res, expected);
 				});
 
-				// Add C6 here
+				mocha.it('C6: 28 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as kWh', async () => {
+					await prepareTest(unitDatakWh, conversionDatakWh, meterDatakWh);
+					const unitId = await getUnitId('kWh');
+					const expected = [108269.924822581, 108889.847659507];
+					const res = await chai.request(app).get('/api/compareReadings/meters/${METER_ID}')
+						.query({
+							curr_start: '2022-10-09 00:00:00',
+							curr_end: '2022-10-31 17:00:00',
+							shift: 'P28D',
```

Comment:
The specified time was '2022-10-31 17:12:34'.

Reply:
Will do, thank you for your prompt feedback!